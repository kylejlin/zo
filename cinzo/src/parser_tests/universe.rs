use super::*;

#[test]
fn type0() {
    let src = r#"Type0"#;
    let tokens = lex(src).unwrap();
    let cst = parse(tokens).unwrap();
    insta::assert_debug_snapshot!(&cst);
}

#[test]
fn type1() {
    let src = r#"Type1"#;
    let tokens = lex(src).unwrap();
    let cst = parse(tokens).unwrap();
    insta::assert_debug_snapshot!(&cst);
}

#[test]
fn type42() {
    let src = r#"Type42"#;
    let tokens = lex(src).unwrap();
    let cst = parse(tokens).unwrap();
    insta::assert_debug_snapshot!(&cst);
}
