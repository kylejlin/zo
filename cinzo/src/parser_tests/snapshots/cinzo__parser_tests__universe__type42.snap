---
source: src/parser_tests/universe.rs
expression: "&cst"
---
Universe(
    UniverseLiteral {
        level: 42,
        start: ByteIndex(
            0,
        ),
    },
)
